src/transformer_deploy/convert.py

```diff
@@ -184,28 +190,35 @@ def main(commands: argparse.Namespace):
 
     tensor_shapes = list(zip(commands.batch_size, commands.seq_len))
     # take optimial size
-    inputs_pytorch = generate_multiple_inputs(
-        batch_size=tensor_shapes[1][0],
-        seq_len=tensor_shapes[1][1],
-        input_names=input_names,
-        device=commands.device,
-        nb_inputs_to_gen=commands.warmup,
-    )
+    if commands.task == "seq2seq":
+        input_ids = tokenizer(
+            "translate English to French: convert this model ...", return_tensors=TensorType.PYTORCH
+        ).input_ids
+        input_ids = input_ids.to("cuda")
+        inputs_pytorch = [{"input_ids": input_ids}]
+    else:
```

Comment:
move this logic inside `generate_multiple_inputs` and add args, so we don't expose those input generation specificities